db/migrate/20240507174646_add_toxic_exposure_to_form526_submissions.rb

```diff
@@ -0,0 +1,5 @@
+class AddToxicExposureToForm526Submissions < ActiveRecord::Migration[7.1]
+  def change
+    add_column :form526_submissions, :toxic_exposure, :boolean, default: false
```

Comment:
feel free to reject this idea

Long term, we aren't going to be evaluating on toxic exposure. Rather than having toxic exposure as a true/false flag, having a `:submission_endpoint` enumeration with `evss` and `lighthouse` could provide insight in the future, as make the backup submission work a bit more straightforward.